nav2_behavior_tree/include/nav2_behavior_tree/random_crawl_action.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_BEHAVIOR_TREE__RANDOM_CRAWL_ACTION_HPP_
+#define NAV2_BEHAVIOR_TREE__RANDOM_CRAWL_ACTION_HPP_
+
+#include <string>
+#include <memory>
```

Comment:
I don't see that memory or cmath are required by this header and should be removed. If the implementation requires them, they should be in the .cpp file.